src/screens/OtherProfile/elements/BioAndChat.js

```diff
@@ -80,7 +79,7 @@ const BioAndChat = (props) => {
     } catch (e) {
       if (e?.response?.data?.status === CHANNEL_BLOCKED) {
         const response = e?.response?.data?.data;
-        await savePendingChatFromOtherProfile(response, true);
+        await savePendingChatFromOtherProfile(response, true, ANON_PM);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change in this line is adding two new parameters to the `savePendingChatFromOtherProfile` function call. However, without seeing the definition of this function, it's hard to say whether these additional parameters are necessary or correctly used. If they are required for the function and their values are correct, then this change is fine.
```diff
-        await savePendingChatFromOtherProfile(response, true);
+        await savePendingChatFromOtherProfile(response, true, ANON_PM);
```
If the function doesn't need these parameters or if the values are incorrect, you should revert this change.


<!-- This is an auto-generated comment by OSS CodeRabbit -->